docker/oap/docker-entrypoint.sh

```diff
@@ -136,7 +136,7 @@ generateStorageMySQL() {
     cat <<EOT >> ${var_application_file}
 storage:
   mysql:
-    metadataQueryMaxSize: \${SW_STORAGE_H2_QUERY_MAX_SIZE:5000}
+    metadataQueryMaxSize: \${SW_STORAGE_MYSQL_QUERY_MAX_SIZE:5000}
```

Comment:
Why this didn't match `application.yml`?  This is the yml generator written by shell, which will be used in container. If you remove other, than there is no chance to override config by system env, like `SW_STORAGE_MYSQL_QUERY_MAX_SIZE`. You should keep recommended config in application.yml and here. Value is configurable, but structure should be here.

Reply:
Hi, It means all the settings here should match with settings for MySQL in `application.yml`?

Reply:
Yes. Because at here, this script is generating that file.